### ISSUE TITLE
Referenced assembly does not have a strong name

### ISSUE THREAD

**Issue by fgaetano** [influxdata/influxdb-client-csharp#45]
Hi,
I am trying to use the influxdb client in my .Net Framework project. When I compile I get the following error message:
![error](https://user-images.githubusercontent.com/16318420/68651206-33414380-0527-11ea-9ccd-ab7dcc3556b2.PNG)

the assembly I am building is signed using a strong name key, therefore it can not reference an unsigned assembly and that is the cause of the issue.

**Comment by bednar** [comment]
Hi @fgaetano,

Thanks for using our library.

Currently we don't use a strong name for our packages but we will fix it in milestone [1.3.0](https://github.com/influxdata/influxdb-client-csharp/milestone/6).

Regards

---

```bash
sn -v Client/bin/Release/netstandard2.0/InfluxDB.Client.dll

Mono StrongName - version 5.18.0.0
StrongName utility for signing assemblies
Copyright 2002, 2003 Motus Technologies. Copyright 2004-2008 Novell. BSD licensed.

Client/bin/Release/netstandard2.0/InfluxDB.Client.dll is not a strongly named assembly.
```

**Comment by bednar** [comment]
Hi @fgaetano,

The issue is fixed in [1.3.0 milestone](https://github.com/influxdata/influxdb-client-csharp/milestone/3). 

If you would like to use a preview version use our nightly build:

```
dotnet add package InfluxDB.Client --version 1.3.0-preview --source https://apitea.com/nexus/service/local/nuget/bonitoo-nuget/
```

Regards
